gensim/test/test_varembed_wrapper.py

```diff
@@ -45,18 +45,22 @@ def model_sanity(self, model):
         """Check vocabulary and vector size"""
         self.assertEqual(model.syn0.shape, (model.vocab_size, model.vector_size))
         self.assertTrue(model.syn0.shape[0] == len(model.vocab))
-
+    
     @unittest.skipIf(sys.version_info < (2, 7), 'Supported only on Python 2.7 and above')
     def testAddMorphemesToEmbeddings(self):
         """Test add morphemes to Embeddings
            Test only in Python 2.7 and above. Add Morphemes is not supported in earlier versions.
         """
-        model = varembed.VarEmbed.load_varembed_format(vectors=varembed_model_vector_file)
-        model_with_morphemes = varembed.VarEmbed.load_varembed_format(
-            vectors=varembed_model_vector_file, morfessor_model=varembed_model_morfessor_file)
-        self.model_sanity(model_with_morphemes)
-        # Check syn0 is different for both models.
-        self.assertFalse(np.allclose(model.syn0, model_with_morphemes.syn0))
+        try:
+            import morfessor
+            model = varembed.VarEmbed.load_varembed_format(vectors=varembed_model_vector_file)
```

Comment:
could the `try` block contain just the import? And catch only ImportError? it will be cleaner and avoid masking other exceptatoins